src/App.tsx

```diff
@@ -1,3 +1,13 @@
-const App = () => <></>;
+import { ChakraProvider } from '@chakra-ui/react';
+
```

Comment:
Lines with imports should be defined one under the other.
Blank lines shouldn't be here.

Reply:
Ok. i wiil change this